app/controllers/podcast_ownerships_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PodcastOwnershipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @podcast_ownership = PodcastOwnership.new
+  end
+
+  def create
+    @podcast_ownership = PodcastOwnership.new(podcast_ownership_params)
```

Comment:
where is `podcast_ownership_params` defined?

Reply:
PodcastOwnership is the join table between users and podcasts so we get the params from there

Reply:
Yeah but the method `podcast_ownership_params` doesn't exist :) I guess you'll have to define it in a similar fashion to https://github.com/forem/forem/blob/910f79e04e402214f33600cddb44367e9c6e5d45/app/controllers/admin/users_controller.rb#L167-L175

but as suggested, I think this should all split in a separate PR :)

Reply:
Okay @rhymes I'll remove the controller and the pundit policy files from this PR. We can keep this PR just for the join table.
Thanks for suggesting that!